aspnetcore/mvc/views/razor.md

```diff
@@ -519,63 +603,34 @@ If "rick@contoso.com" is passed in the model, the view generates the following H
 
 The `@inject` directive enables the Razor Page to inject a service from the [service container](xref:fundamentals/dependency-injection) into a view. For more information, see [Dependency injection into views](xref:mvc/views/dependency-injection).
 
-### @functions
+### @model
```

Comment:
I think it's worth specifying that `@model` is an MVC feature. It doesn't apply to components, or any usage of Razor outside of MVC.